src/strategy/actions/LootRollAction.cpp

```diff
@@ -13,6 +13,30 @@
 #include "PlayerbotAIConfig.h"
 #include "Playerbots.h"
 
+bool LootRollAction::isUseful()
+{
+	Group* group = bot->GetGroup();
+
+	if (!group)
+		return false;
+
+	std::vector<Roll*> rolls = group->GetRolls();
+
+	bool isUseful = false;
+
+	for (Roll*& roll : rolls)
+	{
+		if (roll->playerVote.find(bot->GetGUID())->second == NOT_EMITED_YET)
+		{
+			isUseful = true;
+
+			break;
+		}
+	}
+
+	return isUseful;
+}
+
```

Comment:
This had never been implemented. This means this action was constantly being ran and would eat ticks for no reason on many occasions. This code attempts to exit as quickly as possible to avoid unnecessary actions.
Since we will iterate over rolls, it is useless to call `->isLootRollActive()` because in case it is `false`, then `rolls.size() == 0` and the code will return immediately, otherwise we will need to iterate anyway.